crates/rpc/rpc-convert/src/transaction.rs

```diff
@@ -499,7 +495,7 @@ pub mod op {
             block_env: &BlockEnv,
         ) -> Result<OpTransaction<TxEnv>, Self::Err> {
             Ok(OpTransaction {
-                base: self.try_into_tx_env(cfg_env, block_env)?,
+                base: self.as_ref().clone().try_into_tx_env(cfg_env, block_env)?,
```

Comment:
this will go away when we finalize 

https://github.com/alloy-rs/op-alloy/pull/525/